erpc/evm_block_tracker.go

```diff
@@ -99,6 +99,8 @@ func (e *EvmBlockTracker) LatestBlock() uint64 {
 }
 
 func (e *EvmBlockTracker) FinalizedBlock() uint64 {
+	e.updateMu.Lock()
+	defer e.updateMu.Unlock()
```

Comment:
Prob a RWMutex is better to use here because there might be multiple readers at the same time.